Formula/g/go@1.21.rb

```diff
@@ -0,0 +1,88 @@
+class GoAT121 < Formula
+  desc "Open source programming language to build simple/reliable/efficient software"
+  homepage "https://go.dev/"
+  url "https://go.dev/dl/go1.21.7.src.tar.gz"
+  mirror "https://fossies.org/linux/misc/go1.21.7.src.tar.gz"
```

Comment:
We could remove the `mirror` to get around this
```suggestion
```

Reply:
No idea why it's happening, may just be some network weirdness. I'm just going to merge this after CI finishes and not worry about it.

Reply:
shouldn't be a blocker. I can access the url fine.